### PR TITLE
Add cacheLength to readme

### PR DESCRIPTION
**Pull request by aminya**

### CODE REVIEW

**Comment by sapphi-red** [comment]
@aminya 
Could you add some descriptions about this option somewhere in the README?
Or I will do it.

File: README.md [hunk 1]
```diff
@@ -20,14 +20,13 @@ In tsconfig.json:
   "compilerOptions": {
     // ...
     "plugins": [
-      { "transform": "@sapphi-red/ts-transformer-for-of" }
+      { "transform": "@sapphi-red/ts-transformer-for-of", "cacheLength": false }
```
Comment (aminya) [inline README.md]:
```suggestion
      { 
      	"transform": "@sapphi-red/ts-transformer-for-of", 
      	"cacheLength": false  // cache the length of the array in `for-of` transformations. Defaults to false.
      } 
```

Comment (aminya) [reply README.md]: Does this look good to you?

Comment (sapphi-red) [reply README.md]: Looks good to me. Thank you.

Comment (aminya) [reply README.md]: You're welcome

**Pull request merged**